packaging/common/rabbitmq-server-ha.ocf

```diff
@@ -0,0 +1,1585 @@
+#!/bin/bash
```

Comment:
Why do you need Bash?

After looking at the code, the main Bash feature used is `[[ $var == "literal" ]]`. This is easily replaced with `[ "$var" = "literal" ]`.